synthesizer/dust.py

```diff
@@ -41,10 +41,22 @@ def __init__(self, params={'slope': -1.}):
         slope: float
             power law slope
         """
-        
+
         self.description = 'simple power law dust curve'
         self.params = params
 
+    def tau_x(self, lam):
+        """
+        Calculate optical depth at lam
+
+        Parameters
+        ----------
+        lam: float array
+            wavelength, expected mwith units
```

Comment:
can you expand on what is returned? also typo

Reply:
I'm going to fix this in another branch because I want to rename everything in dust.py as I don't like the current naming convention.